README.md

```diff
@@ -58,6 +58,8 @@ For help getting started with Flutter, view our online
 | markedDateMoreCustomDecoration | `Decoration`    |                                                                                                                   |
 | markedDateMoreCustomTextStyle | `TextStyle`     |                                                                                                                   |
 | headerMargin            | `EdgetInsets`   | `const EdgeInsets.symmetric(vertical: 16.0)`                                                                      |
+| headerTitleTouchable            | `bool`   | `false`|
+| onHeaderTitlePressed            | `Function`   | `() => _selectDateFromPicker()`|
```

Comment:
_selectDateFromPicker could be explained.

Reply:
Well I've tried to put link in it but didn't work in table. I'll just merge this. Thanks.